app/views/queued_videos/index.html.haml

```diff
@@ -4,6 +4,7 @@
       %article
         %header
           %h2 My Queue
+      = form_tag queued_videos_update_path do
```

Comment:
You need the form at the `%header` level. (now its on `%article` level). so push it more to right (along with all tags under `form_tag`).